msticpy/data/core/data_providers.py

```diff
@@ -212,42 +213,6 @@ def connect(self, connection_str: Optional[str] = None, **kwargs):
         logger.info("Adding query pivot functions")
         self._add_pivots(lambda: self._query_time.timespan)
 
-    def exec_query(self, query: str, **kwargs) -> Union[pd.DataFrame, Any]:
```

Comment:
What's the reason for moving this to the mixin class?
No objections, just curious.

Reply:
I moved it here because:
* As the initial empty implementation from *QueryProviderProtocol* of `exec_query` could not be tested, I removed it
* In `_create_split_query_tasks` from *QueryProviderConnectionsMixin*, `exec_query`is invoked thus somethng has to be defined. Between creating an abstract method and using the "real" implementation, I picked the real implementation

I don't mind changing it to something else if needed!